src/main/scala/is/hail/stats/RegressionUtils.scala

```diff
@@ -335,7 +373,7 @@ object RegressionUtils {
   }
 
   //keyedRow consists of row key followed by numeric data (passed with key to avoid copying, key is ignored here)
-  def denseStats(keyedRow: Row, y: DenseVector[Double]): Option[(DenseVector[Double], Double, Double)] = {
+  def statsKeyedRow(keyedRow: Row, y: DenseVector[Double]): Option[(DenseVector[Double], Double, Double)] = {
```

Comment:
This change is completely independent from the logistic change, it shouldn't be in this PR.